src/diffpy/labpdfproc/tests/test_tools.py

```diff
@@ -0,0 +1,30 @@
+from argparse import ArgumentParser
+
+import pytest
+
+from diffpy.labpdfproc.tools import load_additional_info
+
+params5 = [
+    ([], []),
+    ([["weather=rainy"]], [["weather", "rainy"]]),
+    ([["weather=rainy", "day=tuesday"]], [["weather", "rainy"], ["day", "tuesday"]]),
+]
+
+
+@pytest.mark.parametrize("inputs, expected", params5)
+def test_load_additional_info(inputs, expected):
+    actual_parser = ArgumentParser()
+    actual_parser.add_argument("-add", "--additional-info", action="append", metavar=("KEY=VALUE"))
+    actual_args = actual_parser.parse_args([])
+    expected_parser = ArgumentParser()
+    expected_args = expected_parser.parse_args([])
+
+    if not inputs:
```

Comment:
I am not sure why you have conditionals in the test.  Isn't the test that you pass whatever the inputs are to the function and then the function should handle things as you want?

Reply:
Yes thanks for catching that. I changed the codes.